packages/patternfly-3/patternfly-react/src/index.js

```diff
@@ -1,3 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+if (process.env.NODE_ENV !== 'production') {
+  const axe = require('react-axe'); // eslint-disable-line global-require
+  axe(React, ReactDOM, 1000);
+}
+
```

Comment:
I'm not sure about adding this here as it could get included by consumers. What about adding it only to storybook in https://github.com/patternfly/patternfly-react/blob/master/storybook/config.js ?

Reply:
Ah good call, should have thought of that :)